src/app/register/register.component.html

```diff
@@ -0,0 +1,38 @@
+<!--The content below is only a placeholder and can be replaced.-->
+<html>
+  <title> Registration of user.</title>
+  <h1 style="text-align:center; color:darkslateblue;" >
+  User information.
+  </h1>
+  <body>
```

Comment:
Component should have only part which you are going to render inside a index.html. means no need to write html, body and title tag inside your component. more about [SPA](https://en.wikipedia.org/wiki/Single-page_application)